build.sbt

```diff
@@ -70,7 +70,7 @@ val commonSettings = Seq(
   scalacOptions ++= Seq(
     "-feature",
     "-deprecation",
-    "-unchecked",
-    "-Xlint"
+//    "-Xlint",
```

Comment:
I think you can do `-Xlint` followed by the setting you want to disable with a single dash before it: "-Xlint:-adapted-args". I've also seen it combined like this: `-Xlint:_,-adapted-args` (https://github.com/search?utf8=%E2%9C%93&q=language%3Ascala++%22-Xlint%3A_%2C-adapted-args%22+&type=Code&ref=searchresults)